### ISSUE TITLE
[Bug] Accessibility : Vocalise the content when the user change the type of the chips (Name and Description)

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#332]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

Samsung S20

### OS Version

Android 10

### App version

1.0

### Describe the issue

When the user change the type of the chips : the name and the description are updated .
This content change is not vocalised with Talkback.

### Expected Behavior

This content change (Name and description) should be vocalised with Talkback.
